subsys/cpp/CMakeLists.txt

```diff
@@ -1,10 +1,15 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_sources_ifdef(CONFIG_CPLUSPLUS
```

Comment:
Although unrelated to your PR, please remove this as it has no effect and is mis-communicating whether CPLUSPLUS is set or not in this file.

Reply:
I was about asking the same, I will take the change along

Reply:
fixed